### PR TITLE
arch-chroot: add German translation

### PR DESCRIPTION
**Pull request by lemeleti**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/linux/arch-chroot.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# arch-chroot
+
+> Erweiterter `chroot`-Befehl zur Unterstützung des Arch Linux-Installationsprozesses.
+> Weitere Informationen: <https://man.archlinux.org/man/arch-chroot.8>.
+
+- Starte eine interaktive Shell (`bash` ist der Standard) in einem neuen Root-Verzeichnis:
```
Comment (marchersimon) [inline pages.de/linux/arch-chroot.md:6]:
```suggestion
- Starte eine interaktive Shell (standardmäßig `bash`) in einem neuen Root-Verzeichnis:
```
What do you think?

File: pages.de/linux/arch-chroot.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`arch-chroot {{pfad/zum/neuen/root}}`
```
Comment (marchersimon) [inline pages.de/linux/arch-chroot.md:8]:
```suggestion
`arch-chroot {{pfad/zu/neuem/root}}`
```
Please apply this everywhere

File: pages.de/linux/arch-chroot.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Spezifiziere den Benutzer (nicht der jetzige Benutzer) der die Shell ausführt:
+
+`arch-chroot -u {{anderer_benutzer}} {{pfad/zum/neuen/root}}`
+
+- Führe einen benutzerdefinierten Befehl (anstelle des Standardbefehls `bash`) im neuen Root-Verzeichnis aus:
+
+`arch-chroot {{pfad/zum/neuen/root}} {{befehl}} {{befehlsparameter}}`
+
+- Gebe die Shell an, die nicht die Standard-Shell `bash` ist (in diesem Fall sollte das Paket `zsh` auf dem Zielsystem installiert worden sein):
```
Comment (marchersimon) [inline pages.de/linux/arch-chroot.md:18]:
```suggestion
- Gib die Shell an, die nicht die Standard-Shell `bash` ist (in diesem Fall sollte das Paket `zsh` auf dem Zielsystem installiert worden sein):
```

**Comment by marchersimon** [comment]
@lemeleti just a tip: if you create a new branch for every PR (based on main), you get a cleaner commit history. Also you don't run into troubles when you have more than one PR open.

**Comment by lemeleti** [comment]
@marchersimon thanks for the tip :)

File: pages.de/linux/arch-chroot.md [hunk 4]
```diff
@@ -3,18 +3,18 @@
 > Erweiterter `chroot`-Befehl zur Unterstützung des Arch Linux-Installationsprozesses.
 > Weitere Informationen: <https://man.archlinux.org/man/arch-chroot.8>.
 
-- Starte eine interaktive Shell (`bash` ist der Standard) in einem neuen Root-Verzeichnis:
+- Starte eine interaktive Shell (Standardmässig `bash`) in einem neuen Root-Verzeichnis:
```
Comment (marchersimon) [inline pages.de/linux/arch-chroot.md:6]:
```suggestion
- Starte eine interaktive Shell (Standardmäßig `bash`) in einem neuen Root-Verzeichnis:
```
Standardmässig was used in the old spelling. The `ss` got replaced with an `ß`. ([source](https://www.duden.de/rechtschreibung/standardmaeszig))

**Review by marchersimon — APPROVED** [review]
Thanks @lemeleti

**Comment by lemeleti** [comment]
@marchersimon no problem, thanks for the review :D

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
Thanks @lemeleti!

BTW it's not recommendable to make PRs from `main`, make sure you sync your `main` with `tldr-pages:main` before create another PR to avoid conflicts or issues. Let us know if you need any help.

**Pull request merged**